.github/scripts/auto-label.cjs

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const path = require('path');
+
+const babelParser = require('recast/parsers/babel');
+const json5 = require('json5');
+const lodash = require('lodash');
```

Comment:
Do we really need a `lodash` dependency just to use `uniq` and `flatten`? Is it possible to not add this dependency?  I think these are just one line of code each, right?

Reply:
`lodash` is required by `@babel/core` already.